main.go

```diff
@@ -1,25 +1,42 @@
 package main
 
 import (
+	"context"
 	"github.com/ONSdigital/log.go/log"
 	"github.com/cadmiumcat/books-api/api"
 	"github.com/cadmiumcat/books-api/config"
+	"github.com/cadmiumcat/books-api/interfaces"
+	"github.com/cadmiumcat/books-api/mongo"
+	"github.com/gorilla/mux"
 	"os"
 )
 
 const serviceName = "books-api"
 
 func main() {
+	ctx := context.Background()
+
 	log.Namespace = serviceName
 	// Get Config
 	cfg, err := config.Get()
 	if err != nil {
-		log.Event(nil, "error retrieving the configuration", log.FATAL, log.Error(err))
+		log.Event(ctx, "error retrieving the configuration", log.FATAL, log.Error(err))
 		os.Exit(1)
 	}
 
-	log.Event(nil, "loaded configuration", log.INFO, log.Data{"config": cfg})
+	// TODO: figure out why this does not log the binding address for the database
```

Comment:
See previous comment about tags, once you've understood why this is happening. You can delete the TODO line 😃